### ISSUE TITLE
KeePassXC 2.2.2 starts off-screen when switching from dual to single monitor

### ISSUE THREAD

**Issue by stanac** [keepassxreboot/keepassxc#1903]
Some context first. I am using laptop with additional external monitor. External monitor is primary and laptop is located left of the external monitor.

## Expected Behavior
When disconnecting external monitor and starting KeePassXC it's suppose to check if window is off-screen when starting.

## Current Behavior
When disconnecting external monitor and starting KeePassXC it's doesn't check if window is off-screen, so it can happen that window opens with negative value for position on X axis.

## Possible Solution
Removing `MainWindowGeometry` from `keepassxc.ini` resolved the issue.

## Steps to Reproduce (for bugs)
I cannot reproduce is at the moment, I am traveling, however I image it can be reproduced like this:
1. Setup two monitors, make sure right monitor is primary
2. Open the application and move it to left monitor
3. Close the application
4. Disconnect right monitor (so the left is now primary and the only one)
5. Open the application

## Context
No context here, issue occurs when starting the application.

## Debug Info
KeePassXC - Version 2.2.2
Revision: 6d46717

Libraries:
- Qt 5.9.2
- libgcrypt 1.8.1

Operating system: Windows 10 (10.0)
CPU architecture: x86_64
Kernel: winnt 10.0.16299

Enabled extensions:
- KeePassHTTP
- Auto-Type
- YubiKey

Libraries:
- LIBS

## Additional Debug Info
Below is the value from the ini file. After removing `MainWindowGeometry` from the file, application starts at the center of the screen:
```
[GUI]
MainWindowGeometry=@ByteArray(\x1\xd9\xd0\xcb\0\x2\0\0\xff\xff\xfc\xd2\0\0\0\xd8\0\0\0\x1\0\0\x3V\xff\xff\xfc\xda\0\0\0\xf7\xff\xff\xff\xf9\0\0\x3N\0\0\0\0\0\0\0\0\a\x80)
SplitterState=@Invalid()
EntryListColumnSizes=150, 150, 242
EntrySearchColumnSizes=@Invalid()
```

**Comment by droidmonkey** [comment]
Why are you not using 2.3.1?

**Comment by stanac** [comment]
I missed the update on the PC in question, simply forgot.

**Comment by droidmonkey** [comment]
I believe this issue was fixed in 2.3
